communication/src/allocator/zero_copy/push_pull.rs

```diff
@@ -90,7 +90,7 @@ impl<T:Data> Pull<Message<T>> for Puller<T> {
         self.receiver
             .borrow_mut()
             .pop_front()
-            .map(|bytes| unsafe { Message::from_bytes(bytes) });
+            .map(|bytes| Message::from_bytes(bytes));
```

Comment:
```suggestion
            .map(Message::from_bytes);
```

Reply:
Actually, .. I'm not even sure that `zero_copy/` makes any sense without Abomination (certainly `bincode` is not zero copy, so .. does the concept even make sense?{.

Reply:
Tidied, and note left.